platform/security/idp/security-idp-client/src/test/resources/SAMLResponse-malformedAssertion.xml

```diff
@@ -24,4 +24,4 @@
     7C3BreFrsDAfBgNVHSMEGDAWgBThVMeX3wrCv6lfeF47CyvkSBe9xjAgBgNVHREEGTAXgRVsb2Nh
```

Comment:
I'm not sure we can actually change any of these assertions because it is going to invalidate the signatures.